framer/SVGLayer.coffee

```diff
@@ -5,6 +5,13 @@
 {SVGGroup} = require "./SVGGroup"
 {SVGPath} = require "./SVGPath"
 
+denyCopy = ->
+	if Utils.isFramerStudio()
+		throw new Error("SVGLayer does not support the `copy` method")
+	# else
+	console.warn("SVGLayer does not support the `copy` method")
+	return null
```

Comment:
This duplication is not really nice, is there a way to wrap it in a shared function?
Also, I would make the wording a bit more accurate, because we _do_ support it if the layer doesn't have any children, right?